docs/binary-structure.md

```diff
@@ -0,0 +1,176 @@
+# Binary file structure
+
+The log file that Ping Viewer provides has a header description structure that allows program to identify different types of information, the header structure is defined as the following:
+
+- Header (**String**)
+- Log structure version (**int32**)
+- Information about the program that have generated the log
+  - Ping Viewer commit hash (**String**)
+  - Ping Viewer build date (**String**)
+  - Ping Viewer release tag (**String**)
+  - Operating System name (**String**)
+  - Operating System version (**String**)
+- Sensor information
+  - Sensor type (**int32**)
+  - Sensor model (**int32**)
+
+> Note: The binary file structure is done in **big-endian**.
+
+Arrays (**Strings**) are defined with the following structure:
+- Array size (**uint32**)
+- Array data (**byte_array[]**)
+
+The source code that defines this structure can be found in Ping Viewer source code [LogSensorStruct file](https://github.com/bluerobotics/ping-viewer/blob/master/src/link/logsensorstruct.cpp).
+
+## Python example
```

Comment:
suggest committing the example under `/tools` or `/examples` in master, then link it here. (choice is yours)